setup/check_system_requirements.sh

```diff
@@ -356,9 +338,9 @@ if [[ "$1" == "test_data" || "$1" == "test_data_github" ]]; then
             exit 1
         fi
         echo "Installing $test_data_name_string .... "
-        echo "  $test_data_dir/ from $test_data_url via $test_data_source_location"
+        echo "  $test_data_dir/ using $test_data_url via $test_data_source_location"
         if [[ "$test_data_source_location" == "github" ]]; then
-            python $SCRIPT_DIR/download_test_data.py $test_data_url $test_data_dir >> $install_log 2>&1
+            python3 $SCRIPT_DIR/download_test_data.py $test_data_url --output-dir $GEOIPS_TESTDATA_DIR >> $install_log 2>&1
```

Comment:
Is test_data_dir always GEOIPS_TESTDATA_DIR/reponame? Should we remove the test data dir variable for consistency? Is test data dir ever something besides geoips test data dir?

Reply:
Yes, it's set like this in the original script `test_data_dir=$GEOIPS_TESTDATA_DIR/$test_data_name`. It's currently still used by the git clones. We could remove it and download the git repos with the script (which already has the ability to do so)

Reply:
![Screenshot_20240911-134101_GitHub.jpg](https://github.com/user-attachments/assets/33f5d631-2dc5-4318-87b8-2fbbfed034a4)

Isn't it already downloading git repos with the download script?  If yes and test_data_dir is no longer used,  I say remove test data dir variable. If it's still used,  just resolve this conversation,  we can deal with it later,  if we need to.

Reply:
Sorry - yes, you're right. I forgot I did that. Unfortunately still used in a few other small places. I say we deal with later if we need to